client/my-sites/domains/components/form/country-select.jsx

```diff
@@ -16,22 +16,20 @@ import FormInputValidation from 'components/forms/form-input-validation';
 import scrollIntoViewport from 'lib/scroll-into-viewport';
 import FormSelect from 'components/forms/form-select';
 
-export default React.createClass( {
-	displayName: 'CountrySelect',
+export default localize( class extends React.Component {
```

Comment:
Ugh, this made me cringe. I know there was a `export default` here to begin with, but could we change it to this pattern instead?
```
class SomeSuperComponent extends React.Component {
    // ...
}
export default localize( SomeSuperComponent );
```

If we ever need to `connect` something here that will become a PITA. Plus, with the above syntax, you can remove the `displayName` - it will inferred based on the class' name.

Reply:
Same comment for `Input` and `HiddenInput` below.

Reply:
Yep. Consistency is is good! Thanks.